lib/index.js

```diff
@@ -165,9 +165,54 @@ export default Adapter => class PostgreSQLAdapter extends Adapter {
 
     for (let field in options.match) {
       const value = options.match[field]
-      const isArray = fields[field][isArrayKey]
+      const isArray = (fields[field] ?
+              fields[field][isArrayKey] :
+              fields[field.split('_')[0]][isArrayKey])
+      const containsOperator = field.includes('_')
```

Comment:
Actually its not, babel can't transpile things that rely on global prototypes like `Array.prototype`.